hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This patch fixes :func:`~hypothesis.provisional.urls` strategy ensuring that
+`~` (tilde) is treated as one of the url-safe characters (:issue:`2658`).
+
```

Comment:
```suggestion
``~`` (tilde) is treated as one of the url-safe characters (:issue:`2658`).
```

Single-backticks are `monospace` formatting in Markdown, but in ReStructured Text they can have a wide range of meanings - in this case they'll show up as *italics*.  I think this is the last nitpick before I merge though :grin: